examples/one_degree_simulation.jl

```diff
@@ -0,0 +1,254 @@
+# # One-degree global ocean simulation
+#
+# This example shows how to configure a global ocean simulation at 1ᵒ horizontal resolution with
+# realistic bathymetry.
+#
+# For this example, we need Oceananigans, ClimaOcea, and CairoMakie to visualize the simulation.
+# Also CFTime and Dates for date handling.
+
+using ClimaOcean
+using ClimaOcean.ECCO: ECCO4Monthly, NearestNeighborInpainting
+using Oceananigans
+using Oceananigans.Units
+using OrthogonalSphericalShellGrids
+using CFTime
+using Dates
+using Printf
+
+arch = GPU()
+
+# ### Grid and Bathymetry
+
+Nx = 360
+Ny = 180
+Nz = 100
+
+z_faces = exponential_z_faces(; Nz, depth=5000, h=34)
+
+underlying_grid = TripolarGrid(arch;
+                               size = (Nx, Ny, Nz),
+                               z = z_faces,
+                               halo = (5, 5, 4),
+                               first_pole_longitude = 70,
+                               north_poles_latitude = 55)
+
+bottom_height = regrid_bathymetry(underlying_grid;
+                                  minimum_depth = 10,
+                                  interpolation_passes = 75,
+                                  major_basins = 2)
+
+# For this bathymetry at this horizontal resolution we need to manually open the Gibraltar strait.
+tampered_bottom_height = deepcopy(bottom_height)
+view(tampered_bottom_height, 102:103, 124, 1) .= -400
+
+grid = ImmersedBoundaryGrid(underlying_grid, GridFittedBottom(tampered_bottom_height); active_cells_map=true)
+
+# ### Restoring
+
+# We include temperature and salinity surface restoring to ECCO2.
+restoring_rate  = 1 / 10days
+z_below_surface = z_faces[end-1]
+
+mask = LinearlyTaperedPolarMask(southern=(-80, -70), northern=(70, 90), z=(z_below_surface, 0))
+
+dates = DateTimeProlepticGregorian(1993, 1, 1) : Month(1) : DateTimeProlepticGregorian(1993, 11, 1)
```

Comment:
why only up to November here?
the simulation runs for 360 days.. so what happens beyond end of November?

Reply:
right! We need up to 1994 I guess